website/source/docs/agent/services.html.md

```diff
@@ -261,6 +261,46 @@ Multiple services definitions can be provided at once using the plural
 }
 ```
 
+In HCL this would look like:
+
+```hcl
+services = [
+  {
+    id = "red0"
+    name = "redis"
+    tags = [
+      "primary"
+    ]
+    address = ""
+    port = 6000
+    checks = [
+      {
+        args = ["/bin/check_redis", "-p", "6000"]
+        interval = "5s"
+        timeout = "20s"
+      }
+    ]
+  },
+  {
+    id = "red1"
+    name = "redis"
+    tags = [
+      "delayed",
+      "secondary"
+    ]
+    address = ""
+    port = 7000
+    checks = [
+      {
+        args = ["/bin/check_redis", "-p", "7000"]
+        interval = "30s"
+        timeout = "60s"
+      }
+    ]
+  }
+]
+```
```

Comment:
HCL 1 also supports repeated blocks. The following is equivalent to this addition:

```hcl
services {
    id = "red0"
    name = "redis"
    tags = [ "primary" ]
    address = ""
    port = 6000
    checks {
        args = ["/bin/check_redis", "-p", "6000"]
        interval = "5s"
        timeout = "20s"
    }
}
services {
    id = "red1"
    name = "redis"
    tags = [
      "delayed",
      "secondary"
    ]
    address = ""
    port = 7000
    checks {
        args = ["/bin/check_redis", "-p", "7000"]
        interval = "30s"
        timeout = "60s"
    }
  }
```

I think this syntax is nice and I use it personally because I understand the jank - the issue with it is that we have to use plural `services` even though we are representing each service as a single block just because HCL JSON compat uses the same name as an array in JSON. 

I'm not suggesting we should necessarily complicate docs with this but would be interested to see what people think about this.

We should eventually consider moving to HCL2 where we should be able to make block syntax more natural without JSON compat concerns.

Reply:
Oh interesting. I like that better as well. I think it would be okay to use this and add a little disclaimer about why.

Reply:
I'd go for the blocks since they look cleaner and I don't think the plural `services` is a huge problem. Calling out why would probably be enough :)

Reply:
Updated.